File: class06/cpp/AP0601-ClassificacaoIdade.cpp
#include <iostream>
int main() {
    int idade;
    std::cout << "Idade: "; std::cin >> idade;
    if (idade < 12) std::cout << "Crianca" << std::endl;
    else if (idade < 18) std::cout << "Adolescente" << std::endl;
    else if (idade < 60) std::cout << "Adulto" << std::endl;
    else std::cout << "Idoso" << std::endl;
    return 0;
}

File: class06/java/AP0601-ClassificacaoIdade.java
import java.util.Scanner;
public class Idade {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Idade: "); int id = sc.nextInt();
        if (id < 12) System.out.println("Crianca");
        else if (id < 18) System.out.println("Adolescente");
        else if (id < 60) System.out.println("Adulto");
        else System.out.println("Idoso");
    }
}
